earn/src/data/Uniswap.ts

```diff
@@ -8,13 +11,17 @@ import JSBI from 'jsbi';
 import { Token } from 'shared/lib/data/Token';
 import { getToken } from 'shared/lib/data/TokenData';
 import { toBig } from 'shared/lib/util/Numbers';
+import { Address } from 'wagmi';
 
 import UniswapNFTManagerABI from '../assets/abis/UniswapNFTManager.json';
 import UniswapV3PoolABI from '../assets/abis/UniswapV3Pool.json';
 import { convertBigNumbersForReturnContexts } from '../util/Multicall';
 import { UNISWAP_NONFUNGIBLE_POSITION_MANAGER_ADDRESS } from './constants/Addresses';
 import { BIGQ96, Q96 } from './constants/Values';
 
+const FACTORY_ADDRESS = '0x1F98431c8aD98523631AE4a59f267346ea31F984';
```

Comment:
Maybe call this `UNISWAP_FACTORY_ADDRESS` to differentiate it from our factory.

Reply:
This is inside of the uniswap class so I think it is pretty clear. We can update it in the future.